lib/db/database_maker.py

```diff
@@ -0,0 +1,113 @@
+import os.path
+import sqlite3
+
+
+class CreateDatabase():
+
+    __PATH = os.path.abspath(f"{os.curdir}/res/karp.db")
+    __DIR = "res"
+
+    # A private static method to check if the db file exists.
```

Comment:
Could add the @staticmethod decorator to make it static potentially. Either that or the @classmethod decorator. Consult with python docs before deciding on this

Reply:
https://docs.python.org/3/library/functions.html#staticmethod

https://docs.python.org/3/library/functions.html#classmethod